zerver/tests/test_bots.py

```diff
@@ -90,7 +90,7 @@ def test_add_bot_with_bad_username(self) -> None:
         # Invalid username
         bot_info = dict(
             full_name="My bot name",
-            short_name="@",
+            short_name="my\nbot",
```

Comment:
This was testing names that result in invalid email addresses, and the invalidity of `@` is now fixed.